src/ResultSetBuilder.php

```diff
@@ -54,7 +54,7 @@ private function buildModel(Result $result)
 
     public function buildModelFromIndexAndData($indexName, $data)
     {
-        $pureIndexName = IndexBuilder::getPureIndexName($indexName);
```

Comment:
Hum, not a big fan of this change, because "getIndexBuilder" is kind of dumb at the moment:

https://github.com/jolicode/elastically/blob/210f52b7f2f67d19a5f8939a7cc91b60b363761b/src/Client.php#L23-L26

So it's going to boot a new instance of the builder for each search result :fearful:
Maybe we should move the `getPureIndexName` method in the Client?